.github/workflows/deploy.yml

```diff
@@ -36,15 +36,12 @@ jobs:
 
       - uses: actions/setup-python@v5
         with:
-          python-version: 3.12
```

Comment:
As you noted in MM, we'll want to come up with a plan for upgrading this repo to python >=3.10